### PR TITLE
APP_KEY

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *APP_KEY value wasn't provided as requested in https://docs.pixelfed.org/master/configuration.html*

## Solution
- *generate the app_key and indicate in the .env*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
- [ ] **Code review**
- [ ] **Approval (LGTM)**  
*Code review and approval have to be from a member of @YunoHost/apps group*
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/pixelfed_ynh_APP_KEY%20(yalh76)/)

Revision: c37d7903bb9ac0b9c26f31f6c2be87035acaaafb 
Capture

When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -149,8 +149,6 @@ pushd "$final_path"
 	php7.2 artisan horizon:purge
 popd
 
-
-
 ynh_backup_if_checksum_is_different "$final_path/CONFIG_FILE"
```
Comment (Jibec) [inline scripts/upgrade]: I think the .env file should be generated again in the upgrade script. An existing application without the __APP_KEY__ won't use it.

Comment (yalh76) [reply scripts/upgrade]: Done in https://github.com/YunoHost-Apps/pixelfed_ynh/pull/32

Comment (yalh76) [reply scripts/upgrade]: Update done and build OK

**Comment by yalh76** [comment]
PR #27 ready :)

**Pull request merged**